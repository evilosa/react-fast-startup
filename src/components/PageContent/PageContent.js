import React from 'react'
import PropTypes from 'prop-types'

const PageContent = () => {
  return <div>Page content</div>
}

PageContent.propTypes = {
  children: PropTypes.node
}

PageContent.displayName = 'PageContent'

export default PageContent