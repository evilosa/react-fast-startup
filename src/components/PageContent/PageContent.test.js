import React from 'react'
import { shallow } from 'enzyme'
import PageContent from './PageContent'

describe('PageContent', () => {
  const component = shallow(<PageContent/>)

  it('renders correct', () => {
    expect(component).toMatchSnapshot()
  })
})