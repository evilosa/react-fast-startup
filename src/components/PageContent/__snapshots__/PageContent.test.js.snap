// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent renders correct 1`] = `
<div>
  Page content
</div>
`;
